in_text/text_02_drivers.yml

```diff
@@ -454,12 +469,12 @@ entities:
 
 process-date: !expr format(Sys.time(),'%Y%m%d')
 process-description: >-
-  Stream observations and driver data were combined from multiple sources to achieve the most complete temporal coverage possible
+   Stream observations and driver data were combined from multiple sources to achieve the most complete temporal coverage possible
   from 1980-2022. Historical weather driver data from NLDAS, GridMET, and NOAA GEFS (in this release and Oliver et al. 2022) were used to
```

Comment:
Should we add references for NLDAS, GridMET, and NOAA GEFS, as well as the stream driver data?